src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+
+import NRInfraMain from 'images/infrastructure_screenshot-crop_main.webp'
+
+import NRLambdaMain from 'images/lambda_screenshot-crop_main.webp'
+
+import NRLiveArchivesMain from 'images/live-archives_screenshot-crop_main.webp'
+
+import NRLogsMain from 'images/logs_screenshot-crop_main.webp'
+
+import NRK8sMain from 'images/kubernetes_screenshot-crop_main.webp'
+
+import NRMobileMain from 'images/mobile_screenshot-crop_main.webp'
+
+import NRNetworkMain from 'images/network_screenshot-crop_main.webp'
+
+import NROpenTelemetryMain from 'images/opentelemetry_screenshot-crop_main.webp'
+
+import NRServiceLevelsMain from 'images/slm_screenshot-crop_main.webp'
+
+import NRSyntheticMain from 'images/synthetic_screenshot-crop_main.webp'
+
+import NRVulnManagementMain from 'images/vulnerability-management_screenshot-crop_main.webp'
+
+
+Welcome to the New Relic observability platform! Our platform gives you deep visibility into your systems. It helps you solve problems you may not have been aware of, regardless of the technology you use. You can see how New Relic works [here](#how-it-works) or [get started](#get-started) now if you want to jump right in.
+
+New Relic provides many ways to improve visibility. They let you monitor, troubleshoot, and improve digital performance. Here's a brief overview of our most popular capabilities and features.
+
+
+# Alerts [#alerts]
+<SideBySide>
+  <Side>
+Alerts help you identify issues that look unusual in your applications and code. New Relic includes some <InlinePopover type="alerts" /> when you install it, and you can also create your alerts. You receive alerts through tools like PagerDuty, ServiceNow, Jira, Slack, and more. Let us help you know what's critical, manage the suppressed noise, and mitigate alert fatigue. 
+  
+Learn more about alerts [here](/docs/alerts-applied-intelligence/overview/).
+
+  </Side>
+  <Side>
+    <img
+      title="New Relic alerts"
+      alt="New Relic alerts"
+      src={NRAlertsMain}
+    />
+  </Side>
+  </SideBySide>
+
+# APM [#apm]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic APM"
+    alt="New Relic APM"
+    src={NRApmMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic instruments your application by using one of our many language agents. These agents collect metrics from your application and then send the metrics to our <InlinePopover type="apm" />. It lets you monitor your application using pre-built dashboards. 
+  
+  Learn more about APM [here](/docs/apm/new-relic-apm/getting-started/introduction-apm/).
+
+  </Side>
+  </SideBySide>
+
+# Browser [#browser]
+<SideBySide>
+  <Side>
+
+  The <InlinePopover type="browser" /> capability collects data from the navigation timings in all end users' browsers. It sends that data directly to New Relic while blending with the service that served the requests. When you load that page URL, you can easily see which back-end transaction fulfilled that request. 
```

Comment:
```suggestion
  The <InlinePopover type="browser" /> capability collects data from navigation timings in an end user's browser. It sends that data directly to New Relic while blending with the service that served the requests. When you load that page URL, you can easily see which back-end transaction fulfilled that request. 
```

Reply:
I kept it singular to sidestep the plural noun that's possessive awkwardness. But Idk if it matters that much. The back half of this section is 96% there but I think it's missing something.

- I'm not sure what it means that the browser agent "sends data directly to New Relic" (don't all agents/collectors do this?) while "blending with the service that served the rrequest." What does "blending with the service" mean? I think unspooling these two separate ideas would help.
- The last sentence feels important and more closely related to the "blending with the service". 
- "service that served" seems a bit repetitive

Reply:
I rewrote the paragraph, thanks!
  New Relic <InlinePopover type="browser" /> helps you understand website performance and user behavior by monitoring real user data. It tracks page load times, network requests, JavaScript errors, user interactions, and more. Analyzing navigation timing helps you find issues that hurt your web app's performance or backend errors. By making necessary changes based on this analysis, you can optimize the performance of your website.